src/util/Constants.js

```diff
@@ -454,15 +459,21 @@ exports.MessageTypes = [
   null,
   null,
   'REPLY',
+  'APPLICATION_COMMAND',
+  'LOADING',
 ];
 
 /**
  * The types of messages that are `System`. The available types are `MessageTypes` excluding:
  * * DEFAULT
  * * REPLY
+ * * APPLICATION_COMMAND
+ * * LOADING
  * @typedef {string} SystemMessageType
  */
-exports.SystemMessageTypes = exports.MessageTypes.filter(type => type && type !== 'DEFAULT' && type !== 'REPLY');
+exports.SystemMessageTypes = exports.MessageTypes.filter(
+  type => type && type !== 'DEFAULT' && type !== 'REPLY' && type !== 'APPLICATION_COMMAND' && type !== 'LOADING',
```

Comment:
LOADING is a message flag, not a message type

```suggestion
 ];
 
 /**
  * The types of messages that are `System`. The available types are `MessageTypes` excluding:
  * * DEFAULT
  * * REPLY
  * * APPLICATION_COMMAND
  * @typedef {string} SystemMessageType
  */
exports.SystemMessageTypes = exports.MessageTypes.filter(type => type && type !== 'DEFAULT' && type !== 'REPLY');
exports.SystemMessageTypes = exports.MessageTypes.filter(
  type => type && type !== 'DEFAULT' && type !== 'REPLY' && type !== 'APPLICATION_COMMAND',
```